ReleaseProcessAutomation/Git/CommandLineGitClient.cs

```diff
@@ -245,10 +245,11 @@ public string CheckoutNewBranch (string branchName)
     return checkout.Output;
   }
 
-  public void MergeBranch (string branchName, bool noCommit = false)
+  public void MergeBranch (string branchName, bool noCommit = false, bool fixConflicts = false)
   {
     var shouldCommit = noCommit ? "--no-commit" : "";
-    var merge = ExecuteGitCommandWithOutput($"merge {branchName} --no-ff {shouldCommit}");
+    var shouldFixConflicts = fixConflicts ? "-Xtheirs" : "";
+    var merge = ExecuteGitCommandWithOutput($"merge {branchName} --no-ff {shouldCommit} {shouldFixConflicts}");
```

Comment:
Am not sure about the latest, but shouldn't there be more logic now with the merge conflict resolution for added/removed files?